x-pack/plugins/security_solution/common/api/timeline/clean_draft_timelines/clean_draft_timelines_route_schema.yaml

```diff
@@ -0,0 +1,70 @@
+openapi: 3.0.0
+info:
+  title: Elastic Security - Timeline - Draft Timeline API
+  version: 8.9.0
+servers:
+  - url: 'http://{kibana_host}:{port}'
+    variables:
+      kibana_host:
+        default: localhost
+      port:
+        default: '5601'
+paths:
+  /api/timeline/_draft:
+    post:
+      operationId: cleanDraftTimelines
+      summary: Retrieves a draft timeline or timeline template.
+      description: |
+        Retrieves a clean draft timeline. If a draft timeline does not exist, it is created and returned.
+      tags:
+        - access:securitySolution
+      requestBody:
+        description: The type of timeline to create. Valid values are `default` and `template`.
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                timelineType:
+                  $ref: '../model/components.yaml#/components/schemas/TimelineType'
+      responses:
+        200:
+          description: Indicates that the draft timeline was successfully created. In the event the user already has a draft timeline, the existing draft timeline is cleared and returned.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    type: object
+                    properties:
+                      persistTimeline:
+                        type: object
+                        properties:
+                          timeline:
+                            $ref: '../model/components.yaml#/components/schemas/TimelineResponse'
+                required:
+                  - data
+        403:
+          description: Indicates that the user does not have the required permissions to create a draft timeline.
+          content:
+            application:json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                  status_code:
+                    type: number
+        409:
+          description: Indicates that there is already a draft timeline with the given timelineId.
+          content:
+            application:json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                  status_code:
+                    type: number
```

Comment:
I actually think this should be an error because when we call this api endpoint it should create a new timeline or at least return a user's existing draft, otherwise what's the point of calling it? Will add it to our api tech debt